src/OnlineRetailPortal.Web/Models/Money.cs

```diff
@@ -4,10 +4,5 @@ public class Money
     {
         public double Amount { get; set; } 
         public string Currency { get; set; }
-        //public Money(double amount, string currency)
-        //{
```

Comment:
I want to understand why are we sending the money object in response ?
Please loop in people which have worked on this